sympy/physics/mechanics/tests/test_linearize.py

```diff
@@ -131,6 +131,15 @@ def test_linearize_rolling_disc_kane():
     # Check eigenvalues at critical speed are all zero:
     assert sympify(A.subs(upright_nominal).subs(q3d, 1/sqrt(3))).eigenvals() == {0: 8}
 
+    # Check whether alternative solvers work
+    if not USE_SYMENGINE:
```

Comment:
You could also use `@pytest.mark.parametrize`, to run the same function for multiple solvers.

Reply:
I don't think that is necessary, this is sufficient just to know this works. The different methods are checked in the tests for Matrix.solve(), presumably.